ansible/playbooks/roles/kafka/tasks/generate-certificates.yml

```
@@ -1,55 +1,55 @@
 - name: Create stores directory
   file:
-    path: "{{ specification.kafka_var.security.ssl.server.keystore_location | dirname }}"
+    path: "{{ specification.security.ssl.server.keystore_location | dirname }}"
     state: directory
-    owner: "{{ specification.kafka_var.user }}"
-    group: "{{ specification.kafka_var.group }}"
+    owner: "{{ specification.user }}"
+    group: "{{ specification.group }}"
     mode: "0755"
```

Some time ago we agreed to use symbolic modes
```suggestion
    mode: u=rwx,go=rx
```
BTW do we need that wide permissions? (please check also in other places)